frontend/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM docker.io/library/node:20-bookworm-slim AS build
+
+WORKDIR /app
+
+COPY ./yarn.lock /app/yarn.lock
+COPY ./package.json /app/package.json
```

Since we're copying to the `WORKDIR` we can use the shorthand version
```suggestion
COPY ./yarn.lock package.json .
```